client/my-sites/theme/controller.jsx

```diff
@@ -2,43 +2,25 @@
  * External Dependencies
  */
 import React from 'react';
-import omit from 'lodash/omit';
 import debugFactory from 'debug';
 import startsWith from 'lodash/startsWith';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal Dependencies
  */
 import ThemeSheetComponent from './main';
 import ThemeDetailsComponent from 'components/data/theme-details';
 import { getCurrentUser } from 'state/current-user/selectors';
-import { getThemeDetails } from 'state/themes/theme-details/selectors';
 import {
 	receiveThemeDetails,
 	receiveThemeDetailsFailure,
 	setBackPath
 } from 'state/themes/actions';
 import wpcom from 'lib/wp';
 import config from 'config';
-import { decodeEntities } from 'lib/formatting';
 
 const debug = debugFactory( 'calypso:themes' );
-let themeDetailsCache = new Map();
-
-export function makeElement( ThemesComponent, Head, store, props ) {
```

Comment:
`themes/controller.jsx` is (was) using this function: https://github.com/Automattic/wp-calypso/blob/6daec13e0c9adecff3c2cdc5ee9cefd1892e311e/client/my-sites/themes/controller.jsx#L15